docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -303,6 +303,38 @@ Given I start mobile application with capabilities:
 |appium:options.clearSystemFiles|false|
 ----
 
+=== Reinstall mobile application
+
+==== *_Info_*
+
+Remove a mobile application from device and install it again
+
+==== *_Wording_*
+
+[source,gherkin]
+----
+When I reinstall mobile application using bundle identifier `$bundleId`
+----
+
+==== *_Usage_*
+
+.Reinstall mobile application
+[source,gherkin]
+----
+When I reinstall mobile application using bundle identifier `vividus-mobile.app`
+----
+
+One of the key benefit of usage the reinstall step instead of
+[source,gherkin]
+----
+When I close mobile application
+Given I start mobile application with capabilities:
+|name     |value|
+|fullReset|true |
+
+----
+steps pair is ability to avoid device re-allocation on SauceLabs or BrowserStack in case of application re-installation within story run (e.g. in case usage ExamplesTable)
```

Comment:
```suggestion
steps pair is ability to avoid device re-allocation on SauceLabs or BrowserStack in case of application re-installation within story run.
```